austation/code/modules/surgery/organs/tails.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/organ/tail/cat/Insert(mob/living/carbon/human/H, special = 0, drop_if_replaced = TRUE)
+	..()
+	if(istype(H))
+		if(!("tail_human" in H.dna.species.mutant_bodyparts))
+			H.dna.species.mutant_bodyparts |= "tail_human"
+			H.dna.features["tail_human"] = tail_type
+			H.update_body()
+		ADD_TRAIT(H, TRAIT_GRABWEAKNESS, "cat tail grab weakness")
```

Comment:
Since every part of a felinid can be removed, I didn't want to throw on a straight upside to using tails without a downside. Read through my patch notes again, as tails now give you instant getup and immunity to shuttle knockdown

Reply:
If you have any better ideas to balance against instant gettup then do tell